actions.json

```diff
@@ -36,5 +36,7 @@
   "NOTIFICATION_CONTENT_AVAILABLE" : "v1/NOTIFICATION_CONTENT_AVAILABLE",
   "BACKENDSERVICE_HTTP_URI" : "v1/BACKENDSERVICE_HTTP_URI",
   "BACKENDSERVICE_MQTT_CONNECTED" : "v1/BACKENDSERVICE_MQTT_CONNECTED",
-  "SPATIALIO_IMPORT" : "v1/SPATIALIO_IMPORT"
+  "SPATIALIO_IMPORT" : "v1/SPATIALIO_IMPORT",
+  "FETCH_LAYERS" : "v1/FETCH_LAYERS",
+  "CREATE_LAYER" : "v1/CREATE_FEATURE"
```

Comment:
This doesn't match...not sure which one you want.  Also, we already have a create feature action `DATASERVICE_CREATEFEATURE`.